### PR TITLE
Enable BruControl

### PR DESCRIPTION
**Pull request by BrunDog**
Hi, this PR is to enable iSpindel data exchange to BruControl (brewing automation software - brucontrol.com)

### CODE REVIEW

**Comment by thegreatgunbantoad** [comment]
May C++ could be better.  Can this:
>     strcpy(global_name, my_name);
>     strcat(global_name, "_g");
>     sender.add("Name", global_name, 0);
>     sender.add("Value", Gravity, 0);

Be simplified to this, avoiding the need for a temporary variable?
>    sender.add("Name", my_name+"_g", 0);
>    sender.add("Value", Gravity, 0);

Also is there a reason that Name and Value are capitalised, this isn't in keeping with the rest of the code.

**Comment by BrunDog** [comment]
Thanks for reviewing! I don't think you can just add null terminated chars, but maybe I'm wrong. 'Name' and 'Value' are capitalized because that the the field name expected by BruControl (the receiving application).

**Comment by thegreatgunbantoad** [comment]
Cool, always handy to learn new stuff.

**Comment by thegreatgunbantoad** [comment]
I cannot get my noggin around how C++ handles character arrays / strings with so little ability to translate between the two.  I like a heavily typed language but that assumes it contains type casts.  I cant see how you type cast between strings and char arrays at all.

**Comment by thegreatgunbantoad** [comment]
Using 
sender.add("Name", String(my_name) +"_g", 0);
Looks to be allowable in the compiler.  But as I said my C++ is a bit poo.

**Comment by BrunDog** [comment]
Hi @universam1. Wondering the review cycle planning for this PR?

File: pio/lib/WiFiManagerKT/WiFiManagerKT.h [hunk 1]
```diff
@@ -53,7 +53,8 @@ var lAPI = [
 {"name":"Prometheus", "token":0,"server":1,"uri":0,"port":1,"channel":0,"db":0,"username":0,"password":0,"job":1,"instance":1},
 {"name":"MQTT",       "token":0,"server":1,"uri":0,"port":1,"channel":0,"db":0,"username":1,"password":1,"job":0,"instance":0},
 {"name":"ThingSpeak", "token":1,"server":0,"uri":0,"port":0,"channel":1,"db":0,"username":0,"password":0,"job":0,"instance":0},
-{"name":"Blynk",      "token":1,"server":0,"uri":0,"port":0,"channel":0,"db":0,"username":0,"password":0,"job":0,"instance":0}];
+{"name":"Blynk",      "token":1,"server":0,"uri":0,"port":0,"channel":0,"db":0,"username":0,"password":0,"job":0,"instance":0},
+{"name":"BruControl", "token":0,"server":1,"url":1,"port":1,"channel":0,"db":0,"username":0,"password":0,"job":0,"instance":0}];
```
Comment (MarcoCLA) [inline pio/lib/WiFiManagerKT/WiFiManagerKT.h:57]: url->uri

**Review by BrunDog — COMMENTED** [review]
Please complete the pull request. Conflicts resolved.

**Comment by exoticatom** [comment]
It would be great if this is finally resolved. Many of us are waiting for BruControl to be part of releases. 

Thanks!

**Review by universam1 — CHANGES_REQUESTED** [review]
please rebase on latest changes and adjust API usage

**Comment by BrunDog** [comment]
> please rebase on latest changes and adjust API usage

Sorry, I don't know what this means. Can you please clarify?

**Pull request closed without merging**